civiform/civiform
# [Program Card Images] Polish the admin page UI

### Description

This PR makes minor edits to the admin program image upload page to align it closer to [the Figma mocks](https://www.figma.com/file/rsQEvXUCuTQwyECapaKzw9/CiviForm---Image-upload-to-program-cards?type=design&node-id=179-6429&mode=design&t=Ug9DE2f0zwgPoX5E-0). Specifically:
 - Update the title
 - Add some directions about uploading an image
 - Move the image description form to the right side
 - Add placeholder text to the image description form
 - Update wordings on the image description form

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


#### New Features

### Instructions for manual testing

1. Enable the PROGRAM_CARD_IMAGES flag.
2. As an admin, edit a program then click the "Edit program image" button.
3. Verify that the page looks like a close approximation to the Figma mocks. I don't think we're going to get it perfect, but this seems close! Once this is submitted, Ron and Shreya can review and give any additional updates needed.

### Issue(s) this completes

Epic #5676 

Fixes #5999 (sub-task)
